dynawo/sources/Solvers/SolverSIM/DYNSolverSIM.cpp

```diff
@@ -154,6 +144,7 @@ SolverSIM::defineSpecificParameters() {
   parameters_.insert(make_pair("printfl", ParameterSolver("printfl", VAR_TYPE_INT, optional)));
   parameters_.insert(make_pair("optimizeAlgebraicResidualsEvaluations", ParameterSolver("optimizeAlgebraicResidualsEvaluations", VAR_TYPE_BOOL, optional)));
   parameters_.insert(make_pair("skipNRIfInitialGuessOK", ParameterSolver("skipNRIfInitialGuessOK", VAR_TYPE_BOOL, optional)));
+  parameters_.insert(make_pair("minimalAcceptableStep", ParameterSolver("minimalAcceptableStep", VAR_TYPE_DOUBLE, optional)));
```

Comment:
Why do you have this modification here? I believed this parameter was generic ; perhaps a problem with a rebase?

Reply:
minimalAcceptableStep is hardcoed to 0.1 in solverSIM because we use this solver with h=1 but this solver can be used with h=0.1 also or even anything (when I do some tests cases I alaways simulate der(u)=-u and use h=0.01 to integrate in time) so it will block the simulation for no reason and we had no way to change this behavior from a parameter

Reply:
The expected behavior is the following: the default value of solverSim is 0.1, it can be overriden with the SolverImpl parameter

Reply:
(I used it a lot during parallel run, but there might a regression in master?)

Reply:
Ah you are right I did not see it in SolverImpl. I remove it from here